tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch_nightly/requirements.txt

```diff
@@ -1,5 +1,5 @@
 scikit-learn
 packaging==21.3
-transformers==v4.30.0
-accelerate==0.20.1
+transformers==v4.36.0
+accelerate
```

Comment:
Use a fixed version helping increase pipeline's stability with less surprises.

Reply:
Agree, made this change to test the working version as my dev setup was not working at the moment. Later found this: https://github.com/huggingface/transformers/commit/e37ab52dff8ba167938beac8f42349bda3198198 which explains why I saw the first error. Anyways, long story short pinned a version again.